content/automate/ManageIQ/Physical_Infrastructure/PhysicalServer/Provisioning/StateMachines/Methods.class/__methods__/common_provision.rb

```diff
@@ -0,0 +1,17 @@
+#
+# Description: <Method description here>
+#
+$evm.log(:info, "BUREK PROVISIONING COMMON PHYSICAL SERVER")
+
+server = $evm.root["miq_provision_task"].source
+options = $evm.root["miq_provision_task"].options
+
+$evm.log(:info, "server = #{server.id}")
+$evm.log(:info, "server.vendor = #{server.vendor}")
+$evm.log(:info, "server.hostname = #{server.hostname}")
+$evm.log(:info, "server.kind = #{server.kind}")
+
+$evm.log(:info, "options = #{options}")
+$evm.log(:info, "pxe_server_id = #{options[:pxe_server_id]}")
+$evm.log(:info, "src_pxe_image_id = #{options[:src_pxe_image_id]}")
+$evm.log(:info, "src_configuration_profile_id = #{options[:src_configuration_profile_id]}")
```

Comment:
For the newer methods we are switching to a new format which helps us to test the methods easily.

For example you can look at https://github.com/ManageIQ/manageiq-content/blob/2b6360cc885c7f099838c84c121b8fe750ffb8ba/content/automate/ManageIQ/Cloud/VM/Provisioning/Placement.class/__methods__/best_fit_amazon.rb